src/tester.rs

```diff
@@ -346,20 +346,20 @@ pub(crate) enum Status {
 /// A user `TestCmd`.
 #[derive(Clone, Debug)]
 pub(crate) struct TestCmd<'a> {
-    pub status: Option<Status>,
-    pub stderr: Option<Vec<&'a str>>,
-    pub stdout: Option<Vec<&'a str>>,
+    pub status: Status,
+    pub stderr: Vec<&'a str>,
+    pub stdout: Vec<&'a str>,
     /// A list of custom command line arguments which should be passed when
     /// executing the test command.
     pub args: Vec<String>,
 }
 
 impl<'a> TestCmd<'a> {
-    fn default() -> Self {
+    pub fn default() -> Self {
         Self {
-            status: Some(Status::Success),
-            stderr: None,
-            stdout: None,
+            status: Status::Success,
+            stderr: vec!["..."],
```

Comment:
Why not an empty `Vec` here?

Reply:
That would only match the empty string i.e. it would mean that a command that produced any output wouldn't match successfully. I think the more permissive `...` is preferable myself (i.e. match any output, including none at all).

Reply:
Ah yes that makes sense. I agree, by default we should allow any output.